src/components/Product/SessionReplay/index.tsx

```diff
@@ -104,7 +104,7 @@ const subfeatures = [
     },
     {
         title: 'Web or mobile session recording',
-        description: 'Web or Android (beta) available',
+        description: 'Web, iOS, and Android',
```

Comment:
I'm [blocked on getting a local preview](https://posthog.slack.com/archives/C01V9AT7DK4/p1731335716257849) running at the moment, so it may be that @corywatilo wants to consider this in his review, but yes this is what I was thinking

Reply:
Had a minute so I had a bash at adding in a version of this, `<MobileLibraries>`